vendor/engines/saml_authentication/README.md

```diff
@@ -0,0 +1,80 @@
+# SamlAuthentication Engine
+
+NUcore uses [Devise](https://github.com/plataformatec/devise) for authentication,
+and uses the [`saml_authenticatable`](https://github.com/apokalipto/devise_saml_authenticatable)
+module for SAML integration.
+
+## Enabling SAML Authentication
+
+Add the following line to your `Gemfile` if it is not already present:
+
+```ruby
+gem "saml_authentication", path: "vendor/engines/saml_authentication"`
+```
+
+In `config/settings.yml`, uncomment the `saml` section and adjust the settings
+as necessary to connect with your Identity Provider (IdP).
+
+```yaml
+saml:
+  idp_metadata: "https://websso.example.com/idp/metadata"
+  certificate_file: path/to/file.p12
```

Comment:
If we decide to use certificates, it would be nice for the readme to include info on how to generate them.